dataprofiler/profilers/data_labeler_column_profile.py

```diff
@@ -238,6 +239,44 @@ def profile(self):
         }
         return profile
 
+    def diff(self, other_profile, options=None):
+        """
+        Generates the differences between the orders of two DataLabeler columns
+
+        :return: Dict containing the differences between orders in their
+        appropriate output formats
+        :rtype: dict
+        """
+        cls = self.__class__
+        if not isinstance(other_profile, cls):
+            raise TypeError("Unsupported operand type(s) for diff: '{}' "
+                            "and '{}'".format(cls.__name__,
+                                              other_profile.__class__.__name__))
+        
+        labels = self.data_label.split('|')
+
+        avg_preds = self.avg_predictions
+        label_rep = self.label_representation
+        other_labels = other_profile.data_label.split('|')
+        other_avg_preds = other_profile.avg_predictions
+        other_label_rep = other_profile.label_representation
+
+        avg_predictions_diff = dict()
+        label_rep_diff = dict()
+        for key, value in avg_preds.items():
+            avg_predictions_diff[key] = utils.find_diff_of_numbers(
+                value, other_avg_preds[key])
+        for key, value in label_rep.items():
+            label_rep_diff[key] = utils.find_diff_of_numbers(
+                value, other_label_rep[key])
```

Comment:
I don't think we can assume that both will have the same keys can we? People can use the their own data labeler to profile with

Reply:
I think you should iterate through self._possible_data_labels

Reply:
That's a fair point, the two dicts could have varying keys. example:
e.g. 
{'a': 0.4, 'b': 0.2, 'c': 0.4}
{'b': 0.3, 'c': 0.4, 'd': 0.3} 
becomes
{'a': [0.4, None], 'b': 0.1, 'c': 'unchanged', 'd': [None, '0.3]} 
Thoughts?
Can loop through all keys as above and just use None when key is missing.

Reply:
e.g. `self.other_avg_preds.get(key, None)`